config/config.go

```diff
@@ -40,3 +54,20 @@ func Parse(filename string) (Config, error) {
 	dec.UseNumber()
 	return cfg, dec.Decode(&cfg)
 }
+
+// Map converts a Config struct to a map
+func Map(cfg *Config) map[string]interface{} {
+	var res map[string]interface{}
+
+	cfgBytes, err := json.Marshal(cfg)
+	if err != nil {
+		return res
+	}
+
+	err = json.Unmarshal(cfgBytes, &res)
+	if err != nil {
+		return res
```

Comment:
The error should be returned here.

Reply:
we cannot return an error from within a go routine..